yarn.lock

```diff
@@ -2987,20 +2987,6 @@ cozy-harvest-lib@0.38.3:
     react-markdown "4.0.6"
     react-proptypes "1.0.0"
 
-cozy-harvest-lib@0.38.3:
```

Comment:
Sure about that?

Reply:
Yes, it was duplicated here I don't know why :/